src/actions/rootRequestedThunk.tsx

```diff
@@ -23,15 +23,12 @@ export const rootRequestedThunk = () => {
         const handleTheData = (dbrecords: any) => {
             const convert = (dbrecord: IDatabaseRecord) => {
                 return {
-                    parent:         -1,
+                    parent:         dbrecord.childof,
```

Comment:
I think it's better to organize it in such a way that the root node is just another node: less code to maintain and less duplication. This is also why I have advocated to use the same thunk, both for the top node and any other node. 

Maybe a good middle ground is to use ``dbrecord.childof``, but to make sure (in the preprocessing) that the value of the top node's ``.parent`` is a nonsense value like -1.